src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -11,6 +11,14 @@
 * Update activity DisplayName as per the specification.
   ([#5078](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5078))
 
+* Removed reference to `OpenTelemetry` package. This is a **breaking change**
+  for users relying on
+  [SuppressDownstreamInstrumentation](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.GrpcNetClient#suppressdownstreaminstrumentation)
```

Comment:
This should be documented in the Grpc instrumentation CHANGELOG as well as that's where the option is defined.

Reply:
done